webapp/src/main/resources/public/js/components/workbench/TranslationHistoryModal.js

```diff
@@ -19,7 +19,8 @@ class translationHistoryModal extends React.Component {
     renderHistoryItem = (item) => {
         const { textUnit, openTmTextUnitVariantId } = this.props;
         const rowClass = textUnit.getTmTextUnitVariantId() === item.id ? "history-current-variant" : "";
-        const status = item.id && !item.includedInLocalizedFile ? TextUnitSDK.STATUS.REJECTED : item.status;
+        const mtStatus = item.status === TextUnitSDK.STATUS.MACHINE_TRANSLATED || item.status === TextUnitSDK.STATUS.MT_REVIEW_NEEDED;
+        const status = item.id && !item.includedInLocalizedFile && !mtStatus ? TextUnitSDK.STATUS.REJECTED : item.status;
```

Comment:
This logic is a bit complex. 
So a MT translation can never be in the rejected status?

Reply:
It's the other way around, text units in MT states are always not included in the localized files (until they move to APPROVED status). The frontend is defining anything not included in a localized file as rejected, which means any MT state will show as rejected in the history modal if we don't have the above

Reply:
`TRANSLATED_AND_NOT_REJECTED` results in the TextUnitSearcher including variants that have the `included_in_localized_file` column set to True. The AI translations have this set to `false`, it gets set to true in the TextUnitBatchImporter when returned from Smartling, so we're covered on that side. I see now that when we updated to batch inserts the unit tests that validated that false was always passed in that column were updated and that check was removed, will raise a separate PR to re-add that level of verification so that if it changes in future it will cause test failures

Reply:
Unit test PR: https://github.com/pinterest/mojito/pull/183